common/predictive-text/worker/correction/context-tracker.ts

```diff
@@ -423,17 +426,23 @@ namespace correction {
         let token = new TrackedContextToken();
         token.raw = entry;
         if(token.raw) {
-          let tokenTransform = {
-            insert: entry,
-            deleteLeft: 0
-          };
-          // Build a single-entry prob-distribution array... where the single distribution is 100% for the token's actual form.
-          // Basically, assume the token was the correct input, since we lack any actual probability data about the keystrokes
-          // that generated it.
-          token.transformDistributions = [[{
-            sample: tokenTransform,
-            p: 1.0
-          }]];
+          let chars = token.raw.split('');
+
+          /* Build single-entry prob-distribution arrays, one per char... where the single distribution is 100%
+           * for the token's actual characters.  Needed for 14.0 instead of a single token for the whole word
+           * due to current search-algorithm limitations.
+           * Basically, assume the token was the correct input, since we lack any actual probability data
+           * about the keystrokes that generated it.
+           */
+          token.transformDistributions = chars.map(function(char) {
+            return [{
+              sample: {
+                insert: char,
+                deleteLeft: 0
+              },
+              p: 1.0
+            }];
+          });
```

Comment:
This block appears to be the one responsible for build test breakage; reverting it allows the unit tests to pass unaltered.  Not completely surprising b/c of the tests that are breaking:

https://github.com/keymanapp/keyman/blob/58a2bda5344a4fa153f772614d644f30b919ec46/common/predictive-text/unit_tests/headless/worker-model-compositor.js#L178-L194

Line 194 is usually where the most-affected unit test fails.  Note that this unit test starts with a multi-character context; it wasn't being handled super-well before.  That, in turn, affected edit-distance calculations, which then caused a minor domino-style effect... which has illuminated a few other bugs worth addressing.